TestTools/StreamChatTestTools/TestData/AnyEndpoint.swift

```diff
@@ -31,6 +31,14 @@ public struct AnyEndpoint: Equatable {
             && lhs.body == rhs.body
             && lhs.payloadType == rhs.payloadType
     }
+
+    func bodyAsDictionary() throws -> [String: Any] {
+        let data = try JSONEncoder().encode(body)
+        guard let requestBody = try JSONSerialization.jsonObject(with: data) as? [String: Any] else {
```

Comment:
Wondering if you could use `try JSONEncoder().decode([AnyHashable: Any], data: data)`

Reply:
Oh probably yes, but since this is just a workaround it is not a big deal, but we can change it later 👍 This is only needed for some some tests which right now are really hard test due to Tech Debt